docs/source/archive/gsoc/reports/2022/scancodeio_scantext_lf32.rst

```diff
@@ -0,0 +1,90 @@
+=================================================================
+Create a web application to scan and review a single license text
+=================================================================
+
+
+| **Organization:** `AboutCode <https://aboutcode.org>`_
+| **Project:** `Scancode.io <https://github.com/nexB/scancode.io>`_
+| **Mentee:** `Lali Akhil Raj (lf32) <https://github.com/lf32>`_
+| **Mentors:** `Philippe Ombredanne <https://github.com/pombredanne>`_, `Thomas Druez <https://github.com/tdruez>`_, `AyanSinhaMahapatra <https://github.com/AyanSinhaMahapatra>`_, `AvishrantsSh <https://github.com/AvishrantsSh>`_
+
+Overview
+--------
+
+The aim of this project is to create a Django-based web application (scantext)
+to scan and review a single license text inside ScanCode.io.
+The application will be designed around a Django_REST-framework,
+and it will be integrated with ScanCode-Analyzer to automatically
```

Comment:
We have not finished the `scancode-analyzer` part, better leave it out.

Reply:
Sure, will remove it.